daemon/protocol/python/tests/run_many.py

```diff
@@ -12,174 +12,183 @@
 import client_lib as proto
 
 
-class TestManyRunRequests(unittest.TestCase):
```

Comment:
removed the unittest nature from the test - make will not run in Travis if it is a unittest (though make runs locally)